### ISSUE TITLE
Add GitHub Corners to Vocabulary Storybook

### ISSUE THREAD

**Issue by annatuma** [cc-archive/vocabulary-legacy#52]
**Description**
We want to:
1. Add GitHub Corners into the Vocabulary Storybook.
2. Update Invertible to have two knobs: Black on white, or White on black.
3. Keep Corners (they can be left or right).

Related ticket: https://github.com/creativecommons/vue-vocabulary/issues/118

THIS IS BLOCKED BY https://github.com/creativecommons/vocabulary/issues/62

**Comment by annlinros** [comment]
Hi, I am an Outreachy applicant for May 2020. Can I work on this?

**Comment by dhruvkb** [comment]
Yes, sure. Go ahead.

**Comment by Blackcipher101** [comment]
If no one is working on this issue can I work on it @dhruvkb

**Comment by kushaggarwal** [comment]
Hello @dhruvkb 
If this is open I would like to give it a go

**Comment by kushaggarwal** [comment]
@dhruvkb can you please guide as currently there is a MDX in reference for github stories as `github_corners.stories.mdx`
So, here I have to develop another story (Javascript) regarding the same or does it aim towards anything else 
Can you please help
